symbols/overlay29.yml

```diff
@@ -1003,6 +1003,17 @@ overlay29:
         Sets the dungeon PRNG to use the primary LCG for subsequent random number generation.
         
         No params.
+    - name: DungeonMusicIdToMusicId
```

Comment:
"Dungeon music ID" is kind of a confusing name IMO. Can we use the terminology of the `floor_properties` field? Something like `MusicTableIdxToMusicId`?

Reply:
Sure